### DISCUSSION TITLE
vector is not a good name

### DISCUSSION THREAD

**Discussion by ahfuzhang** [vectordotdev/vector#13680] [General]
the name `vector` just too common to search.
why don't use a unique name?
thanks.

**Comment by spencergilbert** [comment]
Hi @ahfuzhang - this has come up in the past, and it's not likely we'll rebrand the project at this stage.

> Thanks for filing this! This discussion has come up before in different forums and so a recorded answer here will be useful.
>
> While we do recognize that the name can make it hard to search as independent search term, it is unlikely that we will change it at this point given we've already built up the branding and a community around the current name that we would harm with a name change. We also don't see the current name as inhibiting Vector's growth. Searching vector along with another term (like log, metric, transform, syslog, etc.) generally does return relevant results.
>
> Similar to Go, with Golang, Rust with Rustlang, we may try to standardize a variant on the name that makes it easier to search as an independent term (like "vector.dev" or "vectordotdev") though. Open to suggestions here 🙂

https://github.com/vectordotdev/vector/issues/11972#issuecomment-1077859352

**Reply by DimDroll** [reply]
I can confirm, if we're talking about google search, for me, searching:
`vector.dev <whatever your query is>`
will return Vector relevant information.
